run-tests.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash -e
+
+set -e
+npm install
+export ACCESS_TOKEN=$(kubectl exec $(get_pod fence) -it -- fence-create token-create --scopes openid,user,fence,data --type access_token --exp 1800 --username cdis.autotest@gmail.com)
```

Comment:
Make the kubectl namespace configurable, and the QA environment URL.
Also need to set the INDEXD_ and EXPIRED_ environment variables?